packages/taiko-client/prover/proof_submitter/proof_buffer_monitor.go

```diff
@@ -9,20 +9,18 @@ import (
 	proofProducer "github.com/taikoxyz/taiko-mono/packages/taiko-client/prover/proof_producer"
 )
 
+const monitorInterval = 5 * time.Minute
```

Comment:
Would it be better if we made it configurable? If there is a chance the we want to change it in the future.